cfgen/cfgen

```diff
@@ -1002,17 +1002,17 @@ def get_lnet_portal_group(
 
 
 def get_libfab_portal_group(
-        hax=22000,
+        hax=22001,
         m0_server=[{'name': 'confd', 'port': m0ProcT.confd},
                    {'name': 'ios', 'port': m0ProcT.ios}],
-        m0_client_s3=22500,
+        m0_client_s3=22501,
         m0_client_other=[]
         ) -> Portals:
     protal_group = Portals(hax=hax, m0_server=m0_server,
```

Comment:
Should read "portal_group" ... etc.

Reply:
@d-nayak I have disabled rounding off, and updated the Port info as per cluster.conf file to fix in LR

Reply:
Hi @pavankrishnat - you need to address the naming of the variable: **protal_group** to **portal_group**.

Reply:
Done